File: tests/fixtures/sample_go_repo/calculator/calculator_test.go
package calculator

import "testing"

func TestAdd(t *testing.T) {
	got := Add(1, 2)
	if got != 3 {
		t.Errorf("Add(1, 2) = %f; want 3", got)
	}
}

func TestSubtract(t *testing.T) {
	got := Subtract(5, 3)
	if got != 2 {
		t.Errorf("Subtract(5, 3) = %f; want 2", got)
	}
}

File: tests/fixtures/sample_java_repo/src/test/java/calculator/CalculatorTest.java
package calculator;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

/**
 * Tests for the Calculator class.
 */
public class CalculatorTest {

    @Test
    public void testAdd() {
        Calculator calc = new Calculator();
        assertEquals(5.0, calc.add(2, 3));
    }

    @Test
    public void testSubtract() {
        Calculator calc = new Calculator();
        assertEquals(2.0, calc.subtract(5, 3));
    }

    @Test
    public void testLastResult() {
        Calculator calc = new Calculator();
        calc.add(3, 4);
        assertEquals(7.0, calc.getLastResult());
    }
}
